tools/git2pantheon/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Is this necessary? Don't we already have a license in the project root that would cover this?

Reply:
Removing it.